x-pack/plugin/watcher/src/main/java/org/elasticsearch/xpack/watcher/execution/ExecutionService.java

```diff
@@ -363,7 +364,8 @@ public void updateWatchStatus(Watch watch) throws IOException {
 
         UpdateRequest updateRequest = new UpdateRequest(Watch.INDEX, Watch.DOC_TYPE, watch.id());
         updateRequest.doc(source);
-        boolean useSeqNoForCAS = clusterService.state().nodes().getMinNodeVersion().onOrAfter(Version.V_6_7_0);
+        boolean useSeqNoForCAS = clusterService.state().nodes().getMinNodeVersion().onOrAfter(Version.V_6_7_0)
```

Comment:
> clusterService.state().nodes().getMinNodeVersion().onOrAfter(Version.V_6_7_0)

I prefer this as a helper method but we already have one with 6.6 (https://github.com/elastic/elasticsearch/blob/6.8/server/src/main/java/org/elasticsearch/action/DocWriteRequest.java#L258). I think it's better to leave it as is.